src/Components/Landing.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+
+import logo from '../Assets/twitter_flipped.png';
+import phone from '../Assets/iphone_mockup.png';
+
+const Landing = () => {
+
+    const [ menu, setMenu ] = useState(false);
+
+    const show = (menu) ? "show" : "" 
```

Comment:
Why the `()` around `menu`?

Reply:
For some reason I was thinking you needed to group the first part of the ternary even if it was one variable, fixed now